### ISSUE TITLE
Item ID typos and spelling errors for batch migration (add yours)

### ISSUE THREAD

**Issue by int-ua** [CleverRaven/Cataclysm-DDA#46289]
# Describe the bug

Some item IDs have errors in them. Items have a way to migrate them to new IDs via `data/json/items/migration.json`.

I suggest we collect all the IDs and migrate them in batch.

ID | suggested replacement
-- | --
balclava | balaclava
lucern_hammer | lucerne_hammer
bowhat | bowlhat, hat_bowl, hat_bowler
porkpie | hat_pork_pie
roller_blades | rollerblades

Vehicle parts, if a migration is implemented:

VP ID | suggested replacement
-- | --
wood box | wood_box
foldable wood box | foldable_wood_box

Bionics

ID | suggested replacement
-- | --
mend_fault_bonic_salvaged_reset | mend_fault_bionic_salvaged_reset

# Additional context

#45878 #44460 #37967 #38073

**Comment by actual-nh** [comment]
Consistent ax vs axe (I prefer the latter):
- ax
- copper_ax
- hand_axe
- makeshift_axe
- primitive_axe
- bowling_axe
- throwing_axe
- crash_axe
- fire_ax
- iceaxe
- pickaxe

**Comment by hexagonrecursion** [comment]
"wood box" and "folding wood box" are not items. They are vehicle parts. Is migration implemented for vehicle parts?

**Comment by Mom-Bun** [comment]
Why?
Players don't see the ID, this is merely dev surface, they don't need to be consistent or spelled correctly if you will only see it if you are debugging or something

**Comment by hexagonrecursion** [comment]
tldr: developers, tileset makers and modders are people too. inconsistent ids cause confusion.

https://github.com/CleverRaven/Cataclysm-DDA/pull/46692#issuecomment-758652238

> > I'm really opposed to this idea. This is churn for churn's sake, and probably breaks the effective DPS tests. There's no reason to change the item IDs for the various axes or the Lucerne Hammer - it doesn't matter if they're not spelled consistently or if `lucern_hammer` isn't spelled "correctly". It's an arbitrary token.
> 
> I understand your feelings but there is a reason: it will decrease possibility of misinterpretation and errors. Take a look at this commit: [I-am-Erk/CDDA-Tilesets@24b9fae#diff-f9149e03f38263415f1e50300769b1b1ba54e84e990f7deae2014b2cb7ee50d4](https://github.com/I-am-Erk/CDDA-Tilesets/commit/24b9fae181d51db2e96e24cff98c4c015446add1#diff-f9149e03f38263415f1e50300769b1b1ba54e84e990f7deae2014b2cb7ee50d4) Lucerne hammer had wrong ID in Ultica until I wrote the detection in the Tracker spreadsheet.
> 
> Another IDs I'd change are `mag_*` books as `mag_pistol` and `mag_smg` were misinterpreted as weapon magazines at least once: [I-am-Erk/CDDA-Tilesets#251 (comment)](https://github.com/I-am-Erk/CDDA-Tilesets/pull/251#issuecomment-690134047)

**Comment by actual-nh** [comment]
In data/json/items/faults_bionic.json: `"id": "mend_fault_bonic_salvaged_reset",`

**Comment by int-ua** [comment]
`fema_entrance` and `FEMA_entrance` are different OMTs https://discord.com/channels/598523535169945603/598614717799596055/920002564718723163
this pair alone blocks ability to rely exclusively on filenames for storing all OMT sprites as on filesystems that treat both cases as the same letter they are the same filename

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
